yabeda-rails.gemspec

```
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.4"
 
   spec.add_dependency "activesupport"
-  spec.add_dependency "anyway_config", ">= 1.3", "< 3"
+  spec.add_dependency "anyway_config", ">= 1.3", "< 2.5.0"
```

Let's rollback this change. I believe that this is some regression that will be fixed soon and we shouldn't care about it as it is not specific to this gem
```suggestion
spec.add_dependency "anyway_config", ">= 1.3", "< 3"
```

---

Fixed here, `.initialize!` sets `Rails.application`:
```diff
--- a/spec/support/rails_app.rb
+++ b/spec/support/rails_app.rb
@@ -32,6 +32,4 @@ class HelloController < ActionController::API
   end
 end
 
-Rails.application = TestApplication
-
 TestApplication.initialize!
```